ui/app/mainui/Popups.qml

```diff
@@ -893,7 +893,7 @@ QtObject {
                     target: root
                     onOwnershipDeclined: {
                         finalisePopup.close()
-                        root.communityTokensStore.ownershipDeclined(communityId)
+                        root.rootStore.communityTokensStore.ownershipDeclined(communityId, communityName)
```

Comment:
I don't like the idea of passing the `communityName` here bc we involve store into doing some UI-specific logic, unrelated to exposing backend to UI (which is the real responsibility of the sotre). If we really need to do it this way for now, I would opt for at lest mentioning it here https://github.com/status-im/status-desktop/issues/12661 as sth to be improved.

Reply:
I'll create a separate task for refactoring:
- The `store` params, only passing `communityId`.
- Backend should emit a signal whenever the decline action happens / is done.
- `ToastsManager` will listen to this new signal and will emit toast according to it.

Reply:
https://github.com/status-im/status-desktop/issues/12709